Makefile

```diff
@@ -0,0 +1,9 @@
+# Use a standard bash shell, avoid zsh or fish
+SHELL:=/bin/bash
+
+# Select the default target, when you are simply running "make"
+.DEFAULT_GOAL:=init
+
+# Include all the targets
+include local/make/dev.mk
+include local/make/lint.mk
```

Comment:
Mega overkill.

Just create venv + `pip install pre-commit`, no need for `pip-tools`, separate requirements file etc.